### ISSUE TITLE
Uncaught (in promise) Error: Unknown layer

### ISSUE THREAD

**Issue by roca77** [tensorflow/tfjs#4390]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow.js installed from (npm or script link): Yes
- TensorFlow.js version: 2.0
- CUDA/cuDNN version:

**Describe the problem**

I am following an image classification tutorial from google Ihttps://www.tensorflow.org/tutorials/images/classification). At some point, there are some data augmentation that are done: **RandomFlip, RandomRotation, RandomZoom.**
When done with the training, I exported the model using tfjs.converters.save_keras_model to be using in my node application.
At node start, I get this error:

_**Uncaught (in promise) Error: Unknown layer: RandomFlip. This may be due to one of the following reasons:
1. The layer is defined in Python, in which case it needs to be ported to TensorFlow.js or your JavaScript code.
2. The custom layer is defined in JavaScript, but is not registered properly with tf.serialization.registerClass().**_

I read somewhere that I could edit the model.json file in some ways to make it work. But I don't know how to apply it to these three transformations. Here is the first lines of the modej.json file:

`
{"format": "layers-model", "generatedBy": "keras v2.4.0", "convertedBy": "TensorFlow.js Converter v2.4.0", "modelTopology": {"keras_version": "2.4.0", "backend": "tensorflow", "model_config": {"class_name": "Sequential", "config": {"name": "sequential_2", "layers": [{"class_name": "InputLayer", "config": {"batch_input_shape": [null, 200, 200, 3], "dtype": "float32", "sparse": false, "ragged": false, "name": "sequential_1_input"}}, {"class_name": "Sequential", "config": {"name": "sequential_1", "layers": [{"class_name": "InputLayer", "config": {"batch_input_shape": [null, 200, 200, 3], "dtype": "float32", "sparse": false, "ragged": false, "name": "random_flip_input"}}, {"class_name": "RandomFlip", "config": {"name": "random_flip", "trainable": true, "batch_input_shape": [null, 200, 200, 3], "dtype": "float32", "mode": "horizontal", "seed": null}}, {"class_name": "RandomRotation", "config": {"name": "random_rotation", "trainable": true, "dtype": "float32", "factor": 0.1, "fill_mode": "reflect", "interpolation": "bilinear", "seed": null}}, {"class_name": "RandomZoom", "config": {"name": "random_zoom", "trainable": true, "dtype": "float32", "height_factor": 0.1, "width_factor": null, "fill_mode": "reflect", "interpolation": "bilinear", "seed": null}}]}}, {"class_name": "Rescaling", "config": {"name": "rescaling_2", "trainable": true, "dtype": "float32", "scale": 0.00392156862745098, "offset": 0.0}}, {"class_name": "Conv2D", "config": {"name": "conv2d_3", "trainable": true, "dtype": "float32", "filters": 16, "kernel_size": [`

Any way to help with this?

**Comment by rthadur** [comment]
@roca77 we will not be able to support "RandomFlip" layer at this time , please let me know if you would like to contribute in adding this layer ?

**Comment by roca77** [comment]
@rthadur Hey, that's a good idea. I can contribute. Where to start?

**Comment by rthadur** [comment]
cc @pyu10055 @lina128

**Comment by rthadur** [comment]
@roca77 here is a previously submitted [Pull Request](https://github.com/tensorflow/tfjs/pull/3702) adding ConvLSTM2D , please refer

**Comment by AnikethYadav** [comment]
hey guys! I too see this error in my piece of code. I am not entirely sure what this error means or how to combat it (i do not have any experience in tf-js)
